metadata.json

```diff
@@ -8,10 +8,72 @@
   "project_page": "https://github.com/puppetlabs/puppetlabs-rook",
   "issues_url": "https://github.com/puppetlabs/puppetlabs-rook/issues",
   "dependencies": [
-    {"name":"puppetlabs-stdlib","version_requirement":">= 4.24.0"},
-    {"name":"puppetlabs/translate","version_requirement":">= 0.0.1 < 1.0.0"},
-    {"name":"stahnma/epel","version_requirement":">= 1.2.2 < 1.3.0"}
+    {
+      "name": "puppetlabs-stdlib",
+      "version_requirement": ">= 4.24.0 < 5.0.0"
```

Comment:
Stdlib 5.0.0 was released so I'd consider bumping this boundary to '< 6.0.0', also Translate was released so you could bump the upper boundary to 2.0.0.

Reply:
@HelenCampbell Thank you for the valuable comment.

Reply:
@HelenCampbell Thanks incorporated the review comments.